lua/wikis/brawlstars/Infobox/Person/Player/Custom.lua

```diff
@@ -65,8 +45,7 @@ function CustomPlayer.run(frame)
 			romanizedname = args.romanized_name,
 			status = args.status,
 			type = player.role.personType,
-			role = player.role.variable,
-			role2 = player.role2.variable,
+			roles = player.roles,
```

Comment:
This won't work as the PlayerIntroduction without changing the PlayerIntro component.

Easiest is using same backwards comptability as LPDB to determine role1 and role2